usr/src/uts/common/fs/zfs/sys/dsl_dir.h

```diff
@@ -81,7 +82,8 @@ typedef struct dsl_dir_phys {
 	uint64_t dd_flags;
 	uint64_t dd_used_breakdown[DD_USED_NUM];
 	uint64_t dd_clones; /* dsl_dir objects */
-	uint64_t dd_pad[13]; /* pad out to 256 bytes for good measure */
+	uint64_t dd_keychain_obj; /* DSL Keychain object number */
```

Comment:
It's preferable to not add fields here, and instead add a `DD_FIELD_*` to the "zapified" dsl_dir.  see `dsl_dir_init_fs_ss_count` for an example of how to do this.